integr8ly/integreatly-operator
# bundle generation for managed-tenants-bundles

# Issue link
https://issues.redhat.com/browse/MGDAPI-2909

# What
Move away from pkgman to bundles

# Verification steps
- From this branch, create a new release of rhoam and integreatly, make sure that the CSV replaces field is correct. 
- Create an RC 2 release and make sure that the CSV is still correct.
- Go through OLM flow installation with existing bundle (n) and newly created one n+, then from n+ to n++
